src/parsing.h

```diff
@@ -26,7 +26,7 @@
 #include "shared-constants.h"
 #include "support/colors.h"
 #include "support/utilities.h"
-#include "wasm-printing.h"
+#include "wasm-traversal.h"
```

Comment:
Is this because `wasm-traversal.h` was previously included indirectly through `wasm-printing.h`?

Reply:
Yes, it didn't build without this.